product_docs/docs/edb_plus/38/01_release_notes/10_edbplus38_rel_notes.mdx

```diff
@@ -0,0 +1,10 @@
+---
+navTitle: Version 38.0.0
+title: "Version 38.0.1"
```

Comment:
I believe title in the document should be 38.0.0 in this document and not 38.0.1